### DISCUSSION TITLE
[Feature Request] Allow placeholders in CMI bungeecord messaging across servers.

### DISCUSSION THREAD

**Discussion by AZFire1** [Zrips/CMI#4518] [Feature Requests]
**Description of feature/improvement request:**

---
### Detailed explanation of what it should do, when and in what situation: ###
The ability to use placeholders for messaging across servers with CMI bungeecord. Currently I'd like for player rank prefixes to show up in messages. I've noticed that they only show up in messages if both players that are in the private message are on the same server, if they aren't it gives the literal placeholder rather then the rank to both players. I do have luckperms on both my servers as well as my proxy server so I don't necessarily think that this is a luckperms placeholder issue.
### Basic explanation as to why this feature is needed and/or useful to have: ###
As I stated above, being able to have placeholders be working and usable across servers through messaging could be nice if you were maybe wanting nicknames to go through or, rank prefixes or really anything I suppose.

**Comment by AZFire1** [comment]
It might be something I misconfigured, I'm unsure.. Added onto this, also you can Tab through all player names to message them in the network with "/reply (tab)" but not with "/msg (name)". Not a very big issue.

**Comment by CloudeLecaw** [comment]
[have you installed cmi bungee?](http://www.zrips.net/wp-content/uploads/2020/02/CMIB1.0.0.4.jar)

> CMI Bungee plugin which is required for you to utilize fur capabilities of CMI plugin while running it at bungee network. Without it, only the portal system will be working, but no messaging across servers. Download it and put into BungeeCord servers plugin folder and reload it.

**Comment by AZFire1** [comment]
Yes.
